app/components/candidate_interface/carry_over_inset_text_component.rb

```diff
@@ -0,0 +1,35 @@
+module CandidateInterface
+  class CarryOverInsetTextComponent < ViewComponent::Base
+    include ViewHelper
+
+    def initialize(application_form:)
+      @application_form = application_form
+    end
+
+    def render?
+      @application_form.unsuccessful_and_apply_2_deadline_has_passed?
+    end
+
+    def application_form_academic_cycle
+      RecruitmentCycle.cycle_name(application_form_recruitment_cycle_year)
```

Comment:
Is this consistent with `application_form_academic_cycle` in the interstitial?

Reply:
Woops - yes that needs to be consistent. Will change.